resources/snakefiles/assemble.smk

```diff
@@ -128,8 +129,8 @@ rule metaquast:
                                  assembler=config['assemblers'],
```

Comment:
for metaquast we to only run on a single assembly, so we don't want to expand `assembler` from `config['assemblers']`; instead, we should pass `assembler=wildcards.assembler` and have `assembler` populated via and `expand` statement in `multiqc_metaquast`